app/views/search/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_tag "/search" do %>
```

Comment:
This will be really helpful in the future, maybe we could make a partial out of it?